handlesErrors.js

```diff
@@ -0,0 +1,26 @@
+const handle404nonExistentPath = (req, res, next) => {
```

Comment:
because this has only 3 arguements, it is not error handling middleware. It would therefore be a bit more suited to have this in the app.js to be triggered like a normal controller.